x-pack/plugins/lens/public/indexpattern_datasource/dimension_panel/droppable.ts

```diff
@@ -159,6 +177,59 @@ export function onDrop(props: DatasourceDimensionDropHandlerProps<IndexPatternPr
   const hasData = Object.values(state.layers).some(({ columns }) => columns.length);
   trackUiEvent(hasData ? 'drop_non_empty' : 'drop_empty');
   setState(mergeLayer({ state, layerId, newLayer }));
-
   return true;
+};
+
+export function onDrop(props: DatasourceDimensionDropHandlerProps<IndexPatternPrivateState>) {
```

Comment:
The changes in this file do not change the functionality yet, right?

Reply:
Right!